bootstrap/peer.go

```diff
@@ -85,7 +84,7 @@ type Peer struct {
 }
 
 // New creates a new Bootstrap Node.
-func New(log *zap.Logger, full *identity.FullIdentity, db DB, revDB extensions.RevocationDB, config Config, versionInfo version.Info) (*Peer, error) {
+func New(log *zap.Logger, full *identity.FullIdentity, db DB, config Config, versionInfo version.Info) (*Peer, error) {
```

Comment:
This PR removes change to ensure that databases are constructed outside of peers / uplink.

Reply:
:+1: